PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/pushimage/PushImageRunState.java

```diff
@@ -99,7 +99,8 @@ public ExecutionResult execute(Executor executor, @NotNull ProgramRunner program
                     Files.write(targetDockerfile, content.getBytes());
 
                     // build image
-                    String imageNameWithTag = dataModel.getPrivateRegistryImageSetting().getImageNameWithTag();
+                    PrivateRegistryImageSetting acrInfo = dataModel.getPrivateRegistryImageSetting();
```

Comment:
can we have a method in PrivateRegstryImageSetting to return the final imageWithTag?